ClassLoader.php

```diff
@@ -0,0 +1,20 @@
+<?php
+class ClassLoader
+{
+    public function ClassLoader()
+    {
+        include_once "BaseOutput.php";
+
+        $dir = __DIR__ . "/OutputPlugins";
+        $allFiles = scandir($dir);
```

Comment:
To make the code easier you can use `glob()` to get the files and `foreach` to loop over them.